components/profile-controller/main.go

```diff
@@ -17,8 +17,10 @@ package main
 
 import (
 	"flag"
+	"io/ioutil"
 	"os"
 
+	"github.com/ghodss/yaml"
```

Comment:
nit: shall we use the official yaml package https://github.com/go-yaml/yaml?
I don't think our use-case is very specific or complex.

EDIT: ohh I realized we are directly unmarshalling into a map[string]string, maybe that's an interesting usage. If it's harder to use the official package, I'm fine with this.

Reply:
I am using this package to be consistent with existing file: profile_controller.go. Keeping this for consistency.